### PR TITLE
Document standalone paginator.

### PR DESCRIPTION
**Pull request by markstory**
Include an example of usage in View Cells. Does this look reasonable @ADmad?

Refs cakephp/cakephp#10718

### CODE REVIEW

File: en/views/cells.rst [hunk 1]
```diff
@@ -205,3 +205,47 @@ name will be used.
...
+                $this->request->getQueryParams(),
+                [
+                    // Use a parameterized custom finder.
+                    'finder' => ['favourites' => [$user]]
+
+                    // Use scoped query string parameters.
+                    'scope' => 'favourites'
+                ]
+            );
+            $this->set('favorites', $results);
```
Comment (jeremyharris) [inline en/views/cells.rst]: You used a different spelling of "favorites" on line 237 😄

**Comment by ADmad** [comment]
@markstory Looks good except for the minor issue jeremyharris pointed out. Thanks for taking care of docs updation.

**Comment by ADmad** [comment]
I updated to consistently use the US spelling :stuck_out_tongue:

**Comment by ADmad** [comment]
Can this be merged or do we wait for FR and JA translations?  (Dunno what's the process followed on docs PRs).

**Comment by HavokInspiration** [comment]
No need to wait for translations.
The tags are just here to help us keep track of what's left to be translated.

**Pull request merged**